cranial/messaging/file.py

```diff
@@ -53,12 +52,16 @@ def send(self, address=None, message='', endpoint=None, serde=json,
                 # make sure the path exists for actual local files.
                 if d != '' and '://' not in endpoint:
                     os.makedirs(d, exist_ok=True)
-                self.logfiles[endpoint] = open(endpoint,
-                                               'ab' if append else 'wb')
+                # todo pass to Instantiate File Connector with address and call get
+                #   with endpoint.
+                #   If given a path, break it up into address and endpoint
+                # self.logfiles[endpoint] = open(endpoint, 'ab' if append else 'wb')
+                self.logfiles[endpoint] = FileConnector(endpoint)
 
-            bytes_written = self.logfiles[endpoint].write(
-                message + '\n'.encode('utf-8'))
-            if bytes_written > 0:
+            # todo instead of write use put and pass in path
+            bytes_written = self.logfiles[endpoint].put(
+                message + '\n'.encode('utf-8'), append=append)
+            if bytes_written is True:
```

Comment:
Connector.put just returns a boolean, so `bytes_written` is a misleading variable name now. (I guess it wasnt fully clear before either, unless you know that IO.write returns the number of bytes.)  Maybe just call it `success` ?